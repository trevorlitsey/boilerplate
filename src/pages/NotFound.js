import React from 'react';
import styled from 'styled-components';

import { Layout } from '../components/layout';

const NotFound = ({ location }) => {
	return (
		<Layout location={location}>
			<h1 style={{ textAlign: 'center', marginTop: 50, lineHeight: 1.5 }}>404</h1>
			<h4 style={{ textAlign: 'center' }}>Sorry, the page you're looking for is not here!</h4>
		</Layout>
	)
}

export default NotFound;