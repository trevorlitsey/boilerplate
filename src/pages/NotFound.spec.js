import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe('NotFound', () => {

	it('should render without crashing', () => {
		const wrapper = shallow(<NotFound />);
		expect(wrapper).toMatchSnapshot();
	})

})