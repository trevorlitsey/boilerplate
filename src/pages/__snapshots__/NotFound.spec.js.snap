// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1
          style={
            Object {
              "lineHeight": 1.5,
              "marginTop": 50,
              "textAlign": "center",
            }
          }
        >
          404
        </h1>,
        <h4
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          Sorry, the page you're looking for is not here!
        </h4>,
      ],
      "location": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
          "style": Object {
            "lineHeight": 1.5,
            "marginTop": 50,
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": "404",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sorry, the page you're looking for is not here!",
          "style": Object {
            "textAlign": "center",
          },
        },
        "ref": null,
        "rendered": "Sorry, the page you're looking for is not here!",
        "type": "h4",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1
            style={
              Object {
                "lineHeight": 1.5,
                "marginTop": 50,
                "textAlign": "center",
              }
            }
          >
            404
          </h1>,
          <h4
            style={
              Object {
                "textAlign": "center",
              }
            }
          >
            Sorry, the page you're looking for is not here!
          </h4>,
        ],
        "location": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
            "style": Object {
              "lineHeight": 1.5,
              "marginTop": 50,
              "textAlign": "center",
            },
          },
          "ref": null,
          "rendered": "404",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sorry, the page you're looking for is not here!",
            "style": Object {
              "textAlign": "center",
            },
          },
          "ref": null,
          "rendered": "Sorry, the page you're looking for is not here!",
          "type": "h4",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
